contrib/build/violation_exceptions.txt

```diff
@@ -0,0 +1,47 @@
+API rule violation: names_match,github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1,CommonServicePlanSpec,ServiceInstanceCreateParameterSchema
```

Comment:
Kubernetes codebase has a similar file https://github.com/kubernetes/kubernetes/blob/master/api/api-rules/violation_exceptions.list

Reply:
I was copying the pattern =)